server/src/main/java/org/elasticsearch/common/settings/KeyStoreWrapper.java

```diff
@@ -526,23 +521,17 @@ public synchronized void save(Path configDir, char[] password) throws Exception
     @Override
     public synchronized SecureString getString(String setting) {
```

Comment:
this could be done in a followup, but with the removal of the distinction within the keystore I think it would be best if the keystore only exposed a method to get the bytes and the individual setting types could handle the conversion to SecureString/InputStream